### PR TITLE
design system and other details

### PR DESCRIPTION
**Pull request by ryanatkn**
Let's merge this whenever you review it. Could be now, or tomorrow, whatever works. I'll keep adding and deploying commits in the meantime.

### CODE REVIEW

File: src/onboard/unconsentful/Revertible.svelte [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
-	<p>Great! We'll never bother you about those things again unless our lawyers make us.</p>
+<Markup>
+	<blockquote>
+		👍 Great! We'll never bother you about those things again unless our lawyers make us.
+	</blockquote>
 	<p>
-		Settings are so annoying right! We're hard at work burying them as deep as we can, because we
-		put you the customer* first.
+		Just checking, you accept all of that stuff, right? Our lawyers are making us ask, you know how
+		lawyers are.
```
Comment (greatbacon) [inline src/onboard/unconsentful/Revertible.svelte:15]:
```suggestion
		lawyers are. 🙄
```

File: src/onboard/unconsentful/Revertible.svelte [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
 	</p>
-	<button on:click={() => done()}>continue →</button>
-</Content>
+	<button on:click={() => done()}>I accept →</button>
```
Comment (greatbacon) [inline src/onboard/unconsentful/Revertible.svelte:17]: Would it be too on the nose to put "What choice do I have?" here?

**Review by greatbacon — APPROVED** [review]
:shipit:

File: src/onboard/unconsentful/Revertible.svelte (hunk 2 above)
Comment (ryanatkn) [reply src/onboard/unconsentful/Revertible.svelte:17]: I think a little yeah and doesn't 100% support "revertible", trying to think of variants that work better. Maybe the text swaps when you mouseover, or maybe there's some game designy thing we can do to make the user *want* to undo, somehow.

**Pull request merged**